src/main/java/io/cdap/e2e/pages/actions/BigQueryActions.java

```diff
@@ -28,7 +29,7 @@
  */
 @Deprecated
 public class BigQueryActions {
-  private static final Logger logger = Logger.getLogger(BigQueryActions.class);
+  private static final Logger logger = LogManager.getLogger(BigQueryActions.class);
```

Comment:
why do we need this change? Isn't bumping up log4j version sufficient?

Reply:
instead of using log4j 1.2.x, now log4j api has been used for moving it to 2.x version. For that, classes has been modified

Reply:
Please move to slf4j / logback as this is logging solution that CDAP uses

Reply:
Working on it. 
Somehow even after changing implementation to logback in e2e-framework, google-cloud module is expecting log4j.properties, logback.xml is not working. Looking into the issue

Reply:
You may need to include https://logging.apache.org/log4j/2.x/log4j-to-slf4j/index.html instead of log4j. Check google-cloud project, it should be solving a similar problem

Reply:
added log4j12 exclusions in hbase dependencies under main dependencies section to make logback work, please confirm if that is ok

Reply:
@tivv In google-cloud project, i have added exclusions for lo4j12 in hbase dependencies under main dependency section, to make logback work in that project. Please confirm if this change is acceptable.